queries.sql

```diff
@@ -1,3 +1,29 @@
 /*Queries that provide answers to the questions from all projects.*/
 
 SELECT * from animals WHERE name = 'Luna';
+
+-- Find all animals whose name ends in "mon".
+SELECT * FROM animals WHERE name LIKE '%mon%';
```

Comment:
Please add a query that can find all animals whose name ends with `mon` not start and ends with `mon`.